### PR TITLE
frature/5_add_readme

### PR DESCRIPTION
**Pull request by frbattid**
- Implements issue #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,146 @@
...
+    * [Installation](#installation)
+    * [Unit tests](#unittests)
+* [Configuration](#configuration)
+* [Running](#running)
+* [Administration](#administration)
+* [Usage](#usage)
+* [Reporting issues and contact information](#contact)
+
+##<a name="whatis"></a>What is frb-cosmos-proxy
+frb-cosmos-proxy is a custom Http proxy acting as a Policy Enorcement Point (PEP). This means:
```
Comment (pcoello25) [inline README.md]: Policy Enorcement Point -> Enforcement

File: README.md [hunk 2]
```diff
@@ -1,2 +1,146 @@
...
+
+* frb-cosmos-proxy is deployed before any Cosmos Http service aimed to be protected.
+* frb-cosmos-proxy is designed to authenticate and authorize the requests origianlly sent to the Cosmos Http service.
```
Comment (pcoello25) [inline README.md]: origianlly -> originally

File: README.md [hunk 3]
```diff
@@ -1,2 +1,146 @@
...
+
+[Top](#top)
+
+##<a name="maininstall"></a>Installation
+This is a software written in JavaScript, specifically suited for [Node.js](https://nodejs.org) (<i>JavaScript on the server side</i>). JavaScript is an interpreted programming language thus it is not necessary to compile it nor build any package; having the source code downloaded somewhere in your machine is enough.
+
+###<a name="prerequisites"></a>Prerequisites
+This PER proxy has no sense if an Identity Manager (Keyrock implementation can be found [here](http://catalogue.fiware.org/enablers/identity-management-keyrock)) is not installed. The same applies to [Cosmos](http://catalogue.fiware.org/enablers/bigdata-analysis-cosmos).
+
+As said, frb-cosmos-proxy is a Node.js application, therefore install it from the official [download](https://nodejs.org/download/). An advanced alternative is to install [Node Version Manager](https://github.com/creationix/nvm) (nvm) by creationix/Tim Caswell, whcih will allow you to have several versions of Node.js and switch among them.
```
Comment (pcoello25) [inline README.md]: whcih -> which

File: README.md (hunk 1 above)
Comment (frbattid) [inline README.md]: Fixed in https://github.com/frbattid/frb-cosmos-proxy/pull/6/commits/0245297b8e62847511e0cf61fd23d63f569c3a1c

File: README.md (hunk 2 above)
Comment (frbattid) [inline README.md]: Fixed in https://github.com/frbattid/frb-cosmos-proxy/pull/6/commits/0245297b8e62847511e0cf61fd23d63f569c3a1c

File: README.md (hunk 3 above)
Comment (frbattid) [inline README.md]: Fixed in https://github.com/frbattid/frb-cosmos-proxy/pull/6/commits/0245297b8e62847511e0cf61fd23d63f569c3a1c

**Comment by pcoello25** [comment]
LGTM

**Pull request merged**